python/tvm/meta_schedule/utils.py

```diff
@@ -31,6 +31,107 @@
 from tvm.tir import FloatImm, IntImm
 
 
+def derived_object(cls: type) -> type:
+    """A decorator to register derived subclasses for TVM objects.
+
+    Parameters
+    ----------
+    cls : type
+        The derived class to be registered.
+
+    Returns
+    -------
+    cls : type
+        The decorated TVM object.
+
+    Example
+    -------
+    .. code-block:: python
+
+        @register_object("meta_schedule.PyRunner")
+        class _PyRunner(meta_schedule.Runner):
+            def __init__(self, methods: List[Callable]):
+                self.__init_handle_by_constructor__(_ffi_api.RunnerPyRunner, *methods)
+
+        class PyRunner():
+            _tvm_metadata = {
+                "cls": _PyRunner,
+                "methods": ["run"]
+            }
```

Comment:
"members" => "fields"

```suggestion
                "cls": _PyRunner,
                "fields": [],
                "methods": ["run"],
            }
```